src/components/dashboard/send-comment.svelte

```diff
@@ -1,18 +1,31 @@
 <script>
   import TextField from 'ui/text-field.svelte';
   import Button from 'ui/button.svelte';
+  import { post } from '@/utils/api';
 
   export let users = [];
-  $: {
-    if (users.length) success = false;
-  }
   let success = false;
   let text = null;
+  let error = null;
 
-  const send = () => {
-    text = null;
-    success = true;
-  };
+  async function send() {
+    if(!text) return;
```

Comment:
also i'd love to see `if` being separated from the `(` with a space :p
same thing on line 19